### ISSUE TITLE
Don't Allow Creation of Environment if VM Template is Not Available

### ISSUE THREAD

**Issue by eengelking** [hobbyfarm/hobbyfarm#369]
**What would you like to be added**:
Disable the creation of Environments if VM Template is not available.

**Why is this needed**:
When creating an environment, you can make it through a lot of steps without realizing that you're missing a requirement. This means you have to dump all of your configuration to this point, go and create a VM Template, come back to the Environment, and then re-create everything.

Instead, it would be nice to have a modal or some other note saying something such as "The following dependencies have not yet been met:" with a list of items needed to populate the environment.

**Comment by jggoebel** [comment]
So a solution would be to disable the „new environment“ button until we have a single VMT?

**Comment by eengelking** [comment]
Exactly. Additionally, I think the order of operation in the left menu would also make sense. So, VM Template should be above Environment, etc.
